website/content/docs/discovery/services.mdx

```diff
@@ -109,6 +108,41 @@ example shows all possible fields, but note that only a few are required.
+| --------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------- | ---------------------------- |
+| `id`                  | String value that specifies the service ID. <p>If not specified, the value of the `name` field will be used. </p>Services must have unique IDs per node, so you should specify unique values if the default `name` will conflict with other services.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Value of the `name` parameter | Optional                     |
+| `name`                | Specifies the name of the service. <p>The value for this parameter is used as the ID if the `id` parameter is not specified.</p>We recommend using valid DNS labels for service definition names for compatibility wit external DNSs.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | None                          | Required                     |
```

Comment:
It would be nice to pick either `<p>` or `<br />` for line breaks. Probably `<br />` since you only need one of them.

Reply:
You're right that we should choose one. I like paragraph tags because I think the line spacing is easier to read, but br tags are definitely easier from an authoring perspective. Maybe we should use br tags because of how ugly the table markup gets after checking into GH.

Reply:
+1 `<br />`